charts/argo-cd/README.md

```diff
@@ -356,7 +361,7 @@ server:
 
 ## Prerequisites
 
-- Kubernetes: `>=1.22.0-0`
+- Kubernetes: `>=1.24.0-0`
```

Comment:
Should this match >=1.22 as well until we bump this?

Reply:
The EoL of K8s v1.22 is over, so I think it is ok to update. but whichever is ok. 🙋

Reply:
Either way, as long we keep it compatible with the upstream :smile: